draft-ietf-trans-rfc6962-bis.md

```diff
@@ -212,6 +212,33 @@ interpreted as described in RFC 2119 [RFC2119].
 Data structures are defined according to the conventions laid out in Section 4
 of [RFC5246].
 
+## Major Differences from RFC6962
+
+- The concept of Precertificate Signing Certificate is gone.
+
+- The poison extension in Precertificates is gone.
+
+- Precertificates are CMS objects, not X.509 certificates.
```

Comment:
Might be worth mentioning why we did this.  i.e., to resolve concerns about violating RFC5820's prohibition on duplicate serial numbers.